src/Components/AnsibleTabs/AnsibleTabs.scss

```diff
@@ -5,91 +5,111 @@ section {
 }
 
 .ins-c-install-client-tab {
-    margin-bottom: var(--pf-global--gutter);
-    p, h4, a {
-        padding: 0 var(--pf-global--gutter) 0 var(--pf-global--gutter);
+    .pf-c-tab-content {
+        padding-top: var(--pf-global--spacer--lg);
     }
-    .ins-c-install-client-tab__links {
-        display: inline-flex;
-        justify-content: space-between;
-        width: 75%;
-        a {
-            text-decoration: none;
-        }
+}
+
+.ins-c-install-client-tab__links {
+    display: inline-flex;
+    justify-content: space-between;
+    width: 75%;
+
+    a {
+        text-decoration: none;
     }
-    .ins-c-install-client-tab__section {
-        padding-right: var(--pf-global--gutter);
-        hr {
-            margin-bottom: 0 !important;
+}
+
+.ins-c-install-client-tab__section {
+    padding-right: var(--pf-global--gutter);
+
+    .pf-c-expandable-section__toggle-text {
+        position: relative;
+        right: calc(var(--pf-global--spacer--sm) + var(--pf-global--spacer--xs));
+    }
+
+    .pf-c-expandable-section {
```

Comment:
all this `ExpandableSection` specific sass isn't really necessary with `Accordion`